src/lib/utils/fetchGitHubContributors.ts

```diff
@@ -0,0 +1,42 @@
+import { join } from "path"
+import { GITHUB_AUTH_HEADERS, GITHUB_COMMITS_URL } from "@/lib/constants"
+import type { Author } from "@/lib/interfaces"
+import type { FileContributorsState } from "@/lib/types"
+
+export const fetchGitHubAuthors = async (
+  relativePath: string
+): Promise<FileContributorsState> => {
+  const url = new URL(GITHUB_COMMITS_URL)
+  const CONTENT_DIR = "/src/content/" // TODO: Overriding with Gatsby repo—Remove and import from constants.ts for production
```

Comment:
Could we add this path to the `constants` file as `OLD_CONTENT_DIR`?

Reply:
Yeah that works for me... I think it was set up like that previously, not sure if there was a reason for changing, but I can add that back in so we can better control the migration.